Bottom: 9efc4cfadc1f5c2bc544dd30e74ab6038a6e2b6c
Top:    6f22ade14aba30608aa90be747a62804142d533a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-18 20:55:22 +0000

Make --enable-dbus default


---

diff --git a/configure.ac b/configure.ac
index 26b20cb..c6f27e5 100644
--- a/configure.ac
+++ b/configure.ac
@@ -353,7 +353,13 @@ fi
 AC_MSG_CHECKING([for whether to use DBUS])
 AC_ARG_ENABLE([dbus],
 [  --enable-dbus           Enable DBUS IPC],
-[],[enable_dbus=no])
+[],[
+	case " $with_gui " in
+		*\ gtk\ *)     enable_dbus=yes ;;
+		*\ lesstif\ *) enable_dbus=yes ;;
+		* )            enable_dbus=no ;;
+	esac
+])
 
 AC_MSG_RESULT([$enable_dbus])
 AM_CONDITIONAL(WITH_DBUS, test x$enable_dbus = xyes)
